src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -562,4 +568,14 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
       .parquet(s"$systemPath/index/${IndexConstants.INDEX_VERSION_DIRECTORY_PREFIX}=$version")
       .count()
   }
+
+  private def getFileIdTracker(indexConfig: IndexConfig): FileIdTracker = {
+    val ixLogPath = PathUtils.makeAbsolute(
+      s"$systemPath/${indexConfig.indexName}/${IndexConstants.HYPERSPACE_LOG}/latestStable")
+    val ixLogJson =
```

Comment:
nit: Shall we use "index" instead of "ix" since we are spelling out "index" throughout the code base?